firmware/ventilator-controller-stm32/Core/Inc/Pufferfish/Driver/I2C/HoneywellABP/Device.h

```diff
@@ -32,23 +31,20 @@ class Device : public Testable {
    */
   I2CDeviceStatus read_sample(ABPSample &sample);
 
-  I2CDeviceStatus test() override;
-  I2CDeviceStatus reset() override;
-
  private:
   Pufferfish::HAL::Interfaces::I2CDevice &dev_;
 
+  static const uint8_t status_shift = 6;
+  static const size_t bridge_high = 0;
+  static const size_t bridge_low = 0;
```

Comment:
How many of these do we actually need? Can we remove `bridge_high` and `bridge_low` now that we use `read_bigend`?

Reply:
Yeah we can remove `bridge_high`  and `bridge_low`